scipy/linalg/_linalg_pythran.py

```diff
@@ -0,0 +1,20 @@
+#pythran export _funm_loops(float32[:, :], float32[:, :], int, float32)
+#pythran export _funm_loops(float64[:, :], float64[:, :], int, float64)
+#pythran export _funm_loops(complex64[:, :], complex64[:, :], int, float32)
+#pythran export _funm_loops(complex128[:, :], complex128[:, :], int, float64)
+def _funm_loops(F, T, n, minden):
+    for p in range(1, n):
+        for i in range(1, n - p + 1):
+            j = i + p
+            s = T[i - 1, j - 1] * (F[j - 1, j - 1] - F[i - 1, i - 1])
+            val = (sum(T[i - 1, i:j - 1] * F[i:j - 1, j - 1])
+                   - sum(F[i - 1, i:j - 1] * T[i:j - 1, j - 1]))
```

Comment:
This works to resolve the lint failure. So that it was more obvious that this was just copy-pasted from `_matfuncs.py`, I did:
```suggestion
            ks = slice(i, j-1)
            val = (sum(T[i - 1, ks] * F[ks, j - 1]) - sum(F[i - 1, ks] * T[ks, j - 1]))
```
But personally I'm happy with what you have here; I just wanted to make it obvious that the code is the same except `dot` is replaced with `*` and `sum`.